### PR TITLE
CoC Clarifications

### PR DESCRIPTION
**Pull request by patricksanders**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -577,8 +577,8 @@ humanoid or AI!**
 Our "Code of Conduct" is **"Be Excellent to Each Other"** AKA the Golden Rule.
 Failing that, it is "Do not be an Ass* or we will kick your ass out!".
 
-Asking questions of a speaker during their talk, to get clarity or debate a
-point is NOT being an ass – heckling or haranguing the speaker IS. If you are
+Asking questions of a speaker during Q&A time, to get clarity or debate a
+point, is NOT being an ass – heckling or haranguing the speaker IS. If you are
```
Comment (brimstone) [inline config.toml]: I don't think this comma is necessary, but it's not blocking the merge.

**Review by brimstone — APPROVED** [review]
LGTM

File: config.toml [hunk 2]
```diff
@@ -577,13 +577,18 @@ humanoid or AI!**
...
+Asking questions of a speaker during Q&A time to get clarity or debate a
 point is NOT being an ass – heckling or haranguing the speaker IS. If you are
 not sure, ask, or err on the side of basic decency and common courtesy. If what
 they are doing would not be acceptable to have done to you, your best friend,
 your worst enemy, your sister, niece, daughter, brother, nephew, son, mother,
 father, or any human being, do not let them treat anyone else that way – whether
- you know them or not. **If someone asks you to stop – stop.**
+you know them or not. **If someone asks you to stop – stop.**
+
+BSides Atlanta seeks to ensure no attendee is harassed. This includes, but is
```
Comment (brimstone) [inline config.toml:588]: Are we BSidesATL or BSides Atlanta? We should be consistent.

Comment (patricksanders) [reply config.toml:588]: We've been using `BSides Atlanta` in text and `BSidesATL` as slugs and usernames. The site header has the long-form version, so I went with that here.

**Review by brimstone — APPROVED** [review]
LGTM. Still ok with these changes.

**Comment by brimstone** [comment]
@patricksanders how do you want to resolve this?

**Comment by patricksanders** [comment]
We can close this PR for now, but this is a good topic of discussion for future meetings. It ended up not being an issue in 2019.

**Pull request closed without merging**